api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirPatientDaoImplTest.java

```diff
@@ -352,7 +352,7 @@ public void searchForPatients_shouldSearchForPatientsByState() {
 	@Test
 	public void searchForPatients_shouldSearchForPatientsByPostalCode() {
```

Comment:
Can we add a test after this one to search by country? It should work using "USA" as the country.

Reply:
My target is to have ~90% patch coverage for each PR and to avoid decreases in overall coverage where ever feasible. I know that's more aggressive than most OpenMRS modules, but since this is a new module, it should be an achievable aim.

Reply:
Wow, thanks bar set high.